### DISCUSSION TITLE
Tags / Description

### DISCUSSION THREAD

**Discussion by jamesatkinsdesign** [GetPublii/Publii#1472] [Help] [answered]
Hi there

In Publii i have Tag setup with a description. The Description has a TinyMCE editor.

On both the preview and live the description is wrapped in a <P> tag and not rendered as html. 

How can I disable or override this?

<img width="428" alt="Screenshot 2024-04-07 at 16 21 33" src="https://github.com/GetPublii/Publii/assets/125280516/ce7b00cb-246d-416c-b9a5-edcf0597491e">
<img width="749" alt="Screenshot 2024-04-07 at 16 22 11" src="https://github.com/GetPublii/Publii/assets/125280516/adc1eb52-6237-4226-aa90-811cbf9242a3">

**Comment by bobmitro** [comment, marked as answer]
Your theme or overridden files might be outdated.
Just look at this topic: https://forum.getpublii.com/topic/the-new-tag-author-options-in-publii-0-42-0/

**Reply by jamesatkinsdesign** [reply]
Thanks - that’s fixed it.
